tests/exception/t13115.nim

```diff
@@ -0,0 +1,13 @@
+discard """
+  exitcode: 1
+  targets: "c"
+  matrix: "-d:debug; -d:release"
+  outputsub: '''t13115.nim(13)           t13115
+Error: unhandled exception: This char is'''
+  outputsub: ''' and works fine! [Exception]'''
```

Comment:
@xflywind there's a bug here:
testament should give error when outputsub is set twice (or better, allow it to do the right thing)
see
```nim
      of "outputsub":
        result.outputCheck = ocSubstr
        result.output = strip(e.value)
```

Reply:
I can't use `output`. The behaviour differs handling `\0` in different platforms in testament. Maybe it is the bug of testament.
If only last outputsup matters, the test is still right.

Reply:
my point was that the 1st outputsub you set is silently overwritten so it wouldn't matter what you would've put there.

and yes, it is a bug of testament, fixing in https://github.com/nim-lang/Nim/pull/16166

Reply:
Yeah I mean `hello\0test` differs in `output` spec
In windows
```nim
hello
test
```
In linux like this or so
```nim
hellotest
```
Works fine if running it locally.

Reply:
Like this
```
Failure: reOutputsDiffer
Expected:
t13115.nim(13)           t13115
Error: unhandled exception: This char is `
` and works fine! [Exception]

Gotten:
t13115.nim(13)           t13115
Error: unhandled exception: This char is `` and works fine! [Exception]
```